.circleci/config.yml

```diff
@@ -86,7 +86,7 @@ commands:
             if [[ $CIRCLE_TAG =~ $pat ]]; then
               echo "this is rc version $CIRCLE_TAG"
               rc_num=$(echo $CIRCLE_TAG | cut -d '-' -f 2)
-              sed -i 's/%d.%d.%d/%d.%d.%d~'$rc_num'/' params/version.go
+              sed -i '' 's/%d.%d.%d/%d.%d.%d~'$rc_num'/' params/version.go
```

Comment:
Yes, GNU sed and BSD sed behaves differently.
```
if [[ "$OSTYPE" == "darwin"* ]]; then
  sed -i '' 's/%d.%d.%d/%d.%d.%d~'$rc_num'/' params/version.go
else
  sed -i 's/%d.%d.%d/%d.%d.%d~'$rc_num'/' params/version.go
fi
```

Reply:
Yes, have to check it's darwin or linux. Thank you.